pkg/crc/machine/vsock.go

```diff
@@ -93,6 +93,22 @@ func vsockPorts(preset crcPreset.Preset, ingressHTTPPort, ingressHTTPSPort uint)
 			Remote:   net.JoinHostPort(virtualMachineIP, internalSSHPort),
 		},
 	}
+
+	if preset == crcPreset.Podman || preset == crcPreset.Microshift {
```

Comment:
We explicitly don't want this for the openshift/okd presets?
Apart from this, looks ok to me

Reply:
The more I think about it, looks like we can have it for each preset.